src/Integration.Vsix_2017/Integration.Vsix.csproj

```diff
@@ -25,6 +27,9 @@
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <ReferencedDirectory>..\Integration.Vsix</ReferencedDirectory>
     <VsCommonIdeDirectory>$(VSAPPIDDIR)</VsCommonIdeDirectory>
+    <StartAction>Program</StartAction>
```

Comment:
I find it very convenient to be able to run the extension without additional configuration. Also, how do you keep local modifications on tracked files. This is very easy to forget and have them commited.